workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/PRMForm/components/fields/InputMultipleFiles/InputMultipleFiles.tsx

```diff
@@ -10,13 +10,15 @@ import {useDropzone} from 'react-dropzone';
 
 import LiferayFile from '../../../../../interfaces/liferayFile';
 import MDFClaim from '../../../../../interfaces/mdfClaim';
+import {ResourceName} from '../../../../../services/liferay/object/enum/resourceName';
 import PRMFormik from '../../../../PRMFormik';
 import ListFiles from '../InputMultipleFilesListing/components/ListFiles';
 import PRMFormFieldStateProps from '../common/interfaces/prmFormFieldStateProps';
 import PRMFormMultipleFilesProps from '../common/interfaces/prmFormMultipleFilesProps';
 
 interface IProps {
 	acceptedFilesExtensions: string;
+	fileResourseName: ResourceName;
```

Comment:
`fileResourseName` spelled incorrectly. should be `fileResourceName`. also fix all the other places the name is used

Reply:
Done!